relayer-cli/src/commands.rs

```diff
@@ -140,6 +141,14 @@ impl Configurable<Config> for CliCmd {
             ccfg.memo_prefix.apply_suffix(&suffix);
         }
 
+        // For all commands except for `start` Hermes retries
+        // for a prolonged period of time.
+        if !matches!(self, CliCmd::Start(_)) {
```

Comment:
We may consider being more restrictive here (only some commands, e.g., `create-*`).

MZ: Anything that is interactive could have a separate config parameter, so we avoid the hardcoded value here.